lib/operations/execute_operation.js

```diff
@@ -159,28 +168,13 @@ function executeWithServerSelection(topology, operation, callback) {
 // TODO: This is only supported for unified topology, it should go away once
```

Comment:
Can you update the language of this note to better explain why this method is necessary?

Reply:
Updated, let me know if there's something specific you think we should have here.